app/src/main/java/io/github/dkter/aaaaa/AaaaaInputMethodService.kt

```diff
@@ -47,6 +60,14 @@ class AaaaaInputMethodService:
         inputChar('a')
     }
 
+    override fun onLongA() {
+        longPressThread = newLongPressThread()
+    }
+
+    override fun onReleaseA() {
+        longPressThread?.interrupt()
+    }
+
     override fun onBackspace() {
```

Comment:
A long-press for backspace would also be useful, I think.

Reply:
I think that's outside the scope of this pull request, but I can open another one, if you want me to.